lib/Dialect/FIRRTL/Transforms/LowerTypes.cpp

```diff
@@ -706,13 +706,14 @@ void TypeLoweringVisitor::visitDecl(FExtModuleOp extModule) {
   for (auto attr : extModule->getAttrDictionary())
     // Drop old "portNames", directions, and argument attributes.  These are
     // handled differently below.
-    if (attr.first != "portNames" && attr.first != direction::attrKey &&
+    if (attr.first != "portNames" &&
+        attr.first != FModuleLike::getPortDirectionsAttrName() &&
```

Comment:
This `getPortDirectionsAttrName()` static method looks great! The juxtaposition with `"portNames"` above and `"portAnnotations"` below highly motivates giving those strings the same treatment in a follow-up commit. 😄